bentoml/_internal/frameworks/tensorflow_v2.py

```diff
@@ -49,6 +51,13 @@
 API_VERSION = "v1"
 
 
+@attr.define
+class KerasOptions(ModelOptions):
+    """Options for the Keras model."""
+
+    partial_kwargs: t.Dict[str, t.Any] = attr.field(factory=dict)
+
+
```

Comment:
this should be called `TensorflowOptions`?

Reply:
Yes. Change was uncommitted. Fixed now.